7-0-java-concurrency/7-0-0-hello-threads/src/test/java/HelloThreadsTest.java

```diff
@@ -0,0 +1,107 @@
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.*;
+
+import java.util.Queue;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertSame;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+class HelloThreadsTest {
+
+    private Queue<Integer> concurrentLinkedQueue;
+
+    @BeforeEach
+    void setup() {
+        concurrentLinkedQueue = new ConcurrentLinkedQueue<>();
+    }
+
+    @SneakyThrows
+    @Test
+    @Order(1)
+    void createThread() {
+        Thread runningThread = HelloThreads.createThread(() -> concurrentLinkedQueue.add(5));
+        assertEquals(0, concurrentLinkedQueue.size());
+
+        runningThread.start();
+        runningThread.join();
+
+        verifyContentQueue();
```

Comment:
@tkuzub I don't like this approach with a shared queue. I guess it's better that you create a new instance of a queue in each test if you need and, then use it, and then verify what you want to verify.